src/lesson2/task1/IfElse.kt

```diff
@@ -79,7 +107,32 @@ fun rookOrBishopThreatens(kingX: Int, kingY: Int,
  * прямоугольным (вернуть 1) или тупоугольным (вернуть 2).
  * Если такой треугольник не существует, вернуть -1.
  */
-fun triangleKind(a: Double, b: Double, c: Double): Int = TODO()
+fun sqr(x:Double) :Double=x*x
+
+fun triangleKind(a: Double, b: Double, c: Double): Int {
+    if((a==b)&&(b==c)&&(c==a)) return 0
```

Comment:
У меня возникли сложности с поиском наибольшей стороны, подскажите, пожалуйста, как её найти.12.09.2016, 19:34, "Marat Akhin" <notifications@github.com>:In src/lesson2/task1/IfElse.kt:
> @@ -79,7 +107,32 @@ fun rookOrBishopThreatens(kingX: Int, kingY: Int,
>   * прямоугольным (вернуть 1) или тупоугольным (вернуть 2).
>   * Если такой треугольник не существует, вернуть -1.
>   */
> -fun triangleKind(a: Double, b: Double, c: Double): Int = TODO()
> +fun sqr(x:Double) :Double=x*x
> +
> +fun triangleKind(a: Double, b: Double, c: Double): Int {
> +    if((a==b)&&(b==c)&&(c==a)) return 0

Чтобы не дублировать код, можно сперва найти наибольшую сторону как новую переменную, а затем все считать относительно нее

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.